libs/service-portal/finance-schedule/src/screens/FinanceSchedule/FinanceSchedule.tsx

```diff
@@ -41,10 +41,13 @@ export const GET_FINANCE_PAYMENT_SCHEDULES = gql`
   }
 `
 
-const FinanceSchedule: ServicePortalModuleComponent = () => {
+const FinanceSchedule: ServicePortalModuleComponent = ({ userInfo }) => {
   useNamespaces('sp.finance-schedule')
   const { formatMessage } = useLocale()
 
+  const actor = userInfo.profile.actor
+  const isDelegation = Boolean(actor)
```

Comment:
feels a little fragile, is this the best signal we have that a delegation is happening?